lib/index.ts

```diff
@@ -120,3 +121,20 @@ async function extractTargetFrameworksFromProjectConfig(
     throw new Error(`Extracting target framework failed with error ${err.message}`);
   }
 }
+
+function containsPackageReference(root: string, manifestFilePath: string) {
+  if (!root || !manifestFilePath) {
+    throw new Error('Missing required parameters for extractTargetFrameworksFromFiles()');
+  }
+
+  const manifestFileFullPath = path.resolve(root, manifestFilePath);
+
+  if (!fs.existsSync(manifestFileFullPath)) {
+    throw new Error('No project file found at ' +
+      `location: ${manifestFileFullPath}`);
+  }
+
+  const manifestFileContents = fs.readFileSync(manifestFileFullPath, 'utf-8');
```

Comment:
Reading in file as I assume we just need text representation of this. We're not needing to parse the XML as I understand, as we just want to know true/false and not the exact list of `PackageReference` items.